tests/lint-diff-test/Makefile

```
@@ -1,4 +1,4 @@
-test: clean test-words test-reldir test-guessstrip test-javaexception
+test: test-words test-reldir test-guessstrip test-javaexception
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)